activemq-karaf/src/main/resources/features-core.xml

```diff
@@ -55,6 +56,7 @@
       <bundle dependency="true">mvn:org.apache.servicemix.bundles/org.apache.servicemix.bundles.jasypt-spring31/1.9.2_1</bundle>
       <bundle dependency="true">mvn:org.apache.servicemix.specs/org.apache.servicemix.specs.stax-api-1.0/${servicemix.specs.version}</bundle>
       <bundle dependency="true">mvn:org.apache.servicemix.bundles/org.apache.servicemix.bundles.xpp3/${xpp3-bundle-version}</bundle>
+      <bundle dependency="true">mvn:joda-time/joda-time/${jodatime-bundle-version}</bundle>
```

Comment:
and this

Reply:
This should *not* be added back in. joda-time dep was removed